test/integration/targets/find/tasks/main.yml

```diff
@@ -375,5 +375,66 @@
       - '"{{ remote_tmp_dir_test }}/astest/.hidden.txt" in astest_list'
       - '"checksum" in result.files[0]'
 
+# Test permission error is correctly handled by find module
+- vars:
+    test_dir: /tmp/permission_test
+    test_user_name: unpriv
+  block:
+    - name: make sure the test user is available
+      include_role:
+        name: setup_test_user
```

Comment:
Isn't this unnecessary, since `setup_test_user` is declared as a dependency?

You should be able to set `test_user_name` in `vars/main.yml` so the user setup is automatic.

Reply:
I wouldn't change `test_user_name` though, unless there's some reason the default isn't acceptable for this test.